app/harvesters/loc_harvester.rb

```diff
@@ -0,0 +1,201 @@
+##
+# Harvester for Library of Congress
+# @see krikri::Harvester
+class LocHarvester < Krikri::Harvesters::ApiHarvester
+  include Krikri::Harvester
+
+  DEFAULT_THREAD_COUNT = 20.freeze
+  JSON_PARAM           = '?fo=json'.freeze
+  
+  # @!attribute [r] opts
+  #   @return [Hash<Symbol, Object>] the options for the harvester
+  # @see .expected_opts
+  attr_reader :opts
+
+  ##
+  # @param opts [Hash] options for the harvester
+  #   The options for this harvester should be an array of URIs because LoC is
+  #   specifying sets of records to be harvested.
+  #
+  # Initial collections uris:
+  #   https://www.loc.gov/collections/american-revolutionary-war-maps/ (1,251 items)
+  #   https://www.loc.gov/collections/civil-war-maps/ (1,707 items)
+  #   https://www.loc.gov/collections/panoramic-maps/ (1,433 items)
+  #
+  # @see .expected_opts
+  def initialize(opts = {})
+    opts[:uri] = '' if opts[:api] # don't require a URI if we have api opts
+
+    opts[:name] ||= 'loc'.freeze
+    opts[:api]  ||= {}
+
+    opts[:api][:uris]          = Array.wrap(opts[:api][:uris])
+    opts[:api][:item_base_uri] = 'https://www.loc.gov/item/'.freeze
+    opts[:api][:threads]       = DEFAULT_THREAD_COUNT
```

Comment:
This gets cast down to `@opts` in the API Harvester's initializer (`super`) https://github.com/dpla/KriKri/blob/develop/lib/krikri/harvesters/api_harvester.rb#L26.

I'm pretty sure it's a local parameter, too. we eventually use it at https://github.com/dpla/heidrun/pull/52/files#diff-1a4317d11c81f5bff6982f05e748c4bfR122, but the AsyncURIGetter never sees this.